nixops/logger.py

```diff
@@ -55,7 +55,7 @@ def log_end(self, prefix, msg):
             self._log_file.write(msg + "\n")
             self._log_file.flush()
 
-    def get_logger_for(self, machine_name):
+    def get_logger_for(self, machine_name: str) -> "MachineLogger":
```

Comment:
add:

```
from __future__ import annotations
```

to the  first first line of code (after the encoding line) and this can just be `MachineLogger`.